metrics/web/docs/features/firestore_emulator_for_driver_tests/diagrams/firestore_emulator_sequence_diagram.puml

```diff
@@ -0,0 +1,62 @@
+@startuml firestore_emulator_sequence_diagram
+
+skinparam ParticipantPadding 20
+skinparam BoxPadding 10
+
+actor Bob
+
+participant FlutterWebDriver
+participant ProcessManager
+participant FirestoreEmulatorRunner
+participant FirebaseProcess
+participant FlutterDriveProcessRunner
+participant Application
```

Comment:
```suggestion
participant Application tests
```
Does it make sense since it seems like we are going to configure the Firestore instance from tests?